src/views/tickets/email.php

```diff
@@ -388,7 +388,7 @@
 																<tr>
 																	<td class="ticket-venue-child" valign="top" align="left" width="130" style="padding: 0 10px 0 0 !important; width:130px; margin:0 !important;">
 																		<span style="color:#0a0a0e !important; font-family: 'Helvetica Neue', Helvetica, sans-serif; font-size:13px; display:block; margin-bottom:5px;"><?php echo $venue_name; ?></span>
-																		<<?php echo $venue_address_tag; ?> style="color:#006caa !important; display:block; margin:0; font-family: 'Helvetica Neue', Helvetica, sans-serif; font-size:13px; text-decoration:underline;" <?php if ( 'a' === $venue_address_tag ) { printf( 'href="%s"', $venue_map_url ); } ?>>
+																		<<?php echo $venue_address_tag; ?> style="<?php if ( ! empty( $venue_map_url ) ) { echo 'color:#006caa !important; text-decoration:underline; '; } ?>display:block; margin:0; font-family: 'Helvetica Neue', Helvetica, sans-serif; font-size:13px;" <?php if ( 'a' === $venue_address_tag ) { printf( 'href="%s"', $venue_map_url ); } ?>>
```

Comment:
* Won't the link inherit normal link styling in any case, removing the need for this? If not, can we add the necessary rules to the `<style>` element instead? That would make this line far more legible
* This code is already pretty hard to read (I realize you're just adopting the existing style but I'm really reluctant to compound things by making the line even longer) ... if the last suggestion isn't viable how about forming the style attribute and placing it inside a var a little earlier, then echoing that var?